README.md

```diff
@@ -1,8 +1,46 @@
-
 # Northcoders News API
 
----Must do to connect to databases locally---
+------------------Link to hosted app--------------------------
+
+# Link: https://ahmed-nc-news.herokuapp.com/api/
+
+This link will show you all the api end points you can access
+
+-------------- About the App ---------------------------
+built a small backend server that will allow front end architecture to
+interact with it via Api endpoints to access information stored on the dataBase.
+This app was built using express.js, node js and postgresql.
+
+------Instructions to clone the app-----------------
+
+1. Fork the git hub repository, Link: [https://github.com/akh701/NCBEproject]
+2. Clone this repository to your local machine using this command in `git clone Forked-Repository-Url`
+3. Install all the following App dependencies:
+
+- [express] with minimum version 4.17.2, command: `npm i expres`
```

Comment:
The command `npm install` will install everything in the package.json.

You dont need to install them all with separate commands